src/content/docs/en/guides/endpoints.mdx

```diff
@@ -47,6 +47,8 @@ import type { APIRoute } from "astro";
 export const GET: APIRoute = async ({ params, request }) => {...}
 ```
 
+Note that these endpoints can only be accessed without a trailing slash, regardless of your [`build.trailingSlash`](/en/reference/configuration-reference/#trailingslash) configuration.
```

Comment:
```suggestion
Note that endpoints whose URLs include a file extension (e.g. `src/pages/sitemap.xml.ts`) can only be accessed without a trailing slash (e.g. `/sitemap.xml`), regardless of your [`build.trailingSlash`](/en/reference/configuration-reference/#trailingslash) configuration.
```
I think this is sufficient for the purposes of *this* PR! I don't think we can say "these" here because I don't think this page makes any distinction between routes that will contain file extensions in the URL vs not.